disnake/http.py

```diff
@@ -2002,6 +2003,37 @@ def get_guild_scheduled_event_users(
         )
         return self.request(route, params=params)
 
+    # Welcome screens
+
+    def get_guild_welcome_screen(
+        self, guild_id: Snowflake
+    ) -> Response[welcome_screen.WelcomeScreen]:
+        r = Route("GET", "/guilds/{guild_id}/welcome-screen", guild_id=guild_id)
+        return self.request(r)
+
+    def edit_guild_welcome_screen(
+        self,
+        guild_id: Snowflake,
+        enabled: bool = None,
+        welcome_channels: List[welcome_screen.WelcomeScreenChannel] = None,
+        description: str = None,
+        reason: str = None,
```

Comment:
Don't they have to have `Optional[x]`? So linters don't complain

Reply:
https://github.com/DisnakeDev/disnake/pull/339/commits/cf6aded359eaa078cfdcaa5997022dcd2fa8287b